internal/pkg/id/generate_test.go

```diff
@@ -7,11 +7,18 @@ package id
 import (
 	"testing"
 
+	"github.com/oklog/ulid"
 	"github.com/stretchr/testify/require"
 )
 
 func TestGenerate(t *testing.T) {
-	id, err := Generate()
-	require.NoError(t, err)
-	require.NotNil(t, id)
+	var pre ulid.ULID
+
+	for i := 0; i < 1000; i++ {
+		id, err := Generate()
+		require.NoError(t, err)
+		require.NotNil(t, id)
+		require.NotEqual(t, id, pre)
+		pre = id
+	}
```

Comment:
[Question/Suggestion]

I wonder if it makes sense to have this test. I'm not a expert in maths or cryptography, but here are my 2 cents:
 - the randomness of the UUIDs are determined by `crypto/rand` and  `github.com/oklog/ulid`. We should not need to test them.
 - you're only guaranteeing it does not generates 2 equal UUIDs in sequence, any other combination it'll not catch. E.g. A-B-A
 - I'm not convinced only 1000 are enough runs to prove or guarantee anything.
 
 I believe this test misleads more than it adds. it misleads in the sense it seems to ensure the randomness is good, when I don't believe it does, and it deviates the test from ensuring the implementation of `Generate` function works. I believe a better test would be to ensure the returned value is non nil, that when transformed into string it has got the expected format. That'd ensure that even if the implementation is changed, if the `ID` is redefined, nothing breaks.